### ISSUE TITLE
Consistent errors while typing

### ISSUE THREAD

**Issue by zslabs** [lesshint/atom-linter-lesshint#25]
Hi,
When attempting to use this plugin, there's times when starting new files (or just typing in existing ones) that throws errors similar to this:

```
TypeError: Cannot read property 'line' of undefined
    at Object.node.nodes.forEach (/Users/zach/.atom/packages/linter-lesshint/node_modules/lesshint/lib/linters/single_line_per_property.js:51:68)
    at Array.forEach (native)
    at Object.singleLinePerPropertyLinter [as lint] (/Users/zach/.atom/packages/linter-lesshint/node_modules/lesshint/lib/linters/single_line_per_property.js:42:20)
    at linters.forEach (/Users/zach/.atom/packages/linter-lesshint/node_modules/lesshint/lib/linter.js:112:38)
    at Array.forEach (native)
    at Linter.checkNode (/Users/zach/.atom/packages/linter-lesshint/node_modules/lesshint/lib/linter.js:84:22)
    at ast.root.walk (/Users/zach/.atom/packages/linter-lesshint/node_modules/lesshint/lib/linter.js:202:22)
    at /Users/zach/.atom/packages/linter-lesshint/node_modules/postcss/lib/container.js:148:26
    at Root.each (/Users/zach/.atom/packages/linter-lesshint/node_modules/postcss/lib/container.js:114:22)
    at Root.walk (/Users/zach/.atom/packages/linter-lesshint/node_modules/postcss/lib/container.js:147:21)
    at Linter.lint (/Users/zach/.atom/packages/linter-lesshint/node_modules/lesshint/lib/linter.js:201:27)
    at Lesshint.checkString (/Users/zach/.atom/packages/linter-lesshint/node_modules/lesshint/lib/lesshint.js:110:29)
    at Object.<anonymous> (/Users/zach/.atom/packages/linter-lesshint/index.js:82:39)
    at next (<anonymous>)
    at step (/Users/zach/.atom/packages/linter-lesshint/index.js:9:273)
```

This happens when opening up, but not closing brackets yet, or starting a variable reference, but not completing it. Because these errors are the big, obtrusive Atom ones, it completely covers the editor workspace and makes it very hard to use. Any help would be appreciated. Using latest version of Atom (stable) and this package. Thanks!

**Comment by jwilsson** [comment]
Hi!
Do you have an example of some code that's failing? I can't seem to reproduce it, even when trying with the examples you mentioned.

**Comment by kokarn** [comment]
I've had this problem before but not recently. I haven't written much LESS last few weeks tho.

Can you make sure you are on the latest version?

**Comment by zslabs** [comment]
It would honestly start with something as simple as `.test {` (before I was able to write the ending `}`. I uninstalled and reinstalled and will report back if it pops up again; trying to see if there was some junk files left over from a previous update.

**Comment by jwilsson** [comment]
Cool, let us know if you find anything!

**Comment by zslabs** [comment]
Here's an example. My package is up to date (have reinstalled) and Atom is also up to date.

![panels](https://cloud.githubusercontent.com/assets/1018402/26408940/23f90ae2-406d-11e7-82a7-be61d720c06a.gif)

**Comment by jwilsson** [comment]
@zslabs Thanks! I'm able to reproduce it, it's an issue in `lesshint` though so I'll create a new issue there.

**Comment by zslabs** [comment]
Great, thanks!

**Comment by zslabs** [comment]
👋 There was a recent update to `lesshint` -- any chance that includes a fix for those?

**Comment by jwilsson** [comment]
@zslabs Sorry, no fix for this issue in `lesshint` yet. The corresponding issue is https://github.com/lesshint/lesshint/issues/395 if you got some time over 😀

**Comment by jwilsson** [comment]
@zslabs This should be fixed in `atom-linter-lesshint@4.0.1`.

**Comment by zslabs** [comment]
🎉 Thank you!
